docs/development/manual_tests.md

```diff
@@ -166,3 +166,51 @@ nextflow run ../main.nf -profile test,singularity --outdir ./results -resume -du
 nextflow run ../main.nf -profile test,singularity --outdir ./results -resume -dump-channels -ansi-log false --input data/samplesheet.tsv --fasta data/reference/Mammoth_MT_Krause.fasta --preprocessing_skippairmerging
 
 ```
+
+## Deduplication
+
+### MARKDUPLICATES
+
+#### With FastP (implicit)
+
+```bash
+## MARKDUPLICATES with default parameters
+nextflow run main.nf -profile docker,test --outdir ~/eager_dsl2_testing/markduplicates -dump-channels -ansi-log false -w ~/eager_dsl2_testing/markduplicates/work/ --deduplication_tool markduplicates -resume
+
+## Run only on merged reads
+nextflow run main.nf -profile docker,test --outdir ~/eager_dsl2_testing/markduplicates_merged -dump-channels -ansi-log false -w ~/eager_dsl2_testing/markduplicates_merged/work/ --deduplication_tool markduplicates --preprocessing_excludeunmerged -resume
+```
+
+#### With AdapterRemoval
+
+```bash
+nextflow run main.nf -profile docker,test --outdir ~/eager_dsl2_testing/AR_markduplicates -dump-channels -ansi-log false -w ~/eager_dsl2_testing/AR_markduplicates/work/ --preprocessing_tool 'adapterremoval' --deduplication_tool 'markduplicates' -resume
+
+nextflow run main.nf -profile docker,test --outdir ~/eager_dsl2_testing/AR_markduplicates_merged -dump-channels -ansi-log false -w ~/eager_dsl2_testing/AR_markduplicates_merged/work/ --preprocessing_tool 'adapterremoval' --deduplication_tool 'markduplicates' --preprocessing_excludeunmerged -resume
+```
+
```

Comment:
Please add  for all a new commented line with `Expect:` describing what files should be there, and maybe approximate numbers of reads (For example). Also please remove the custom paths (work not needed at lesat)